### ISSUE TITLE
Error while running wfmash using the higher-sensitivity branch

### ISSUE THREAD

**Issue by srividya22** [waveygang/wfmash#120]
I have been trying to use the higher-sensitivity branch for running wfmash alignments on my data. The code seems to be much faster than the master for producing the alignments although gives me an error on my data. 

Error: 00:01:46:17[wflign::wflign_affine_wavefront] corrupted traceback (M, but there is a mismatch) for SlycHeinz 71480000 Sabu2 90877136

The same data runs without any issues using the wfmash exe from the master branch. I am trying to run it on a chr  of about 11 tomato genomes which are highly divergent ( diverged about ~ 16 Million year ago ) using parameters used are below

[wfmash::map] Reference = [chr3/all_pangenome_chr3.fasta.gz]
[wfmash::map] Query = [chr3/all_pangenome_chr3.fasta.gz]
[wfmash::map] Kmer size = 19
[wfmash::map] Window size = 122
[wfmash::map] Segment length = 5000 (read split allowed)
[wfmash::map] Block length min = 25000
[wfmash::map] Chaining gap max = 250000
[wfmash::map] Percentage identity threshold = 90%
[wfmash::map] Skip self mappings
[wfmash::map] Filter mode = 1 (1 = map, 2 = one-to-one, 3 = none)
[wfmash::map] Execution threads  = 16
[wfmash::skch::Sketch::build] minimizers picked from reference = 18579250
[wfmash::skch::Sketch::index] unique minimizers = 7107944
[wfmash::skch::Sketch::computeFreqHist] Frequency histogram of minimizers = (1, 3754493) ... (7011, 1)
[wfmash::skch::Sketch::computeFreqHist] With threshold 0.5%, ignore minimizers occurring >= 30 times during lookup.
[wfmash::map] time spent computing the reference index: 17.5171 sec
[wfmash::skch::Map::mapQuery] mapped  2.85% @ 6.53e+07 bp/s elapsed: 00:00:00:00 remain: 00:00:[wfmash::skch::Map::mapQuery] mapped  2.85% @ 4.35e+07 bp/s elapsed: 00:00:00:00 remain: 00:00:[wfmash::skch::Map::mapQuery] mapped  7.15% @ 8.18e+07 bp/s elapsed: 00:00:00:01 remain: 00:00:[wfmash::skch::Map::mapQuery] mapped  7.15% @ 6.55e+07 bp/s elapsed: 00:00:00:01 remain: 00:00:[wfmash::skch::Map::mapQuery] mapped  7.15% @ 5.46e+07 bp/s elapsed: 00:00:00:01 remain: 00:00:[wfmash::skch::Map::mapQuery] mapped  7.15% @ 4.68e+07 bp/s elapsed: 00:00:00:01 remain: 00:00:
...
...
...
[wfmash::skch::Map::mapQuery] mapped 89.62% @ 3.04e+07 bp/s elapsed: 00:00:00:33 remain: 00:00:[wfmash::skch::Map::mapQuery] mapped 89.62% @ 3.02e+07 bp/s elapsed: 00:00:00:34 remain: 00:00:[wfmash::skch::Map::mapQuery] mapped 89.62% @ 2.99e+07 bp/s elapsed: 00:00:00:34 remain: 00:00:[wfmash::skch::Map::mapQuery] mapped 100.00% @ 3.32e+07 bp/s elapsed: 00:00:00:34 remain: 00:00:00:00
[wfmash::skch::Map::mapQuery] count of mapped reads = 10, reads qualified for mapping = 11, total input reads = 11, total input bp = 1144534361
[wfmash::map] time spent mapping the query: 3.45e+01 sec
[wfmash::map] mapping results saved in: /home/sramakr4/scr4_mschatz1/sramakr4/assembly/pansol/pangenomes/wfmash-M89rDi
[wfmash::align] Reference = [chr3/all_pangenome_chr3.fasta.gz]
[wfmash::align] Query = [chr3/all_pangenome_chr3.fasta.gz]
[wfmash::align] Mapping file = /home/sramakr4/scr4_mschatz1/sramakr4/assembly/pansol/pangenomes/wfmash-M89rDi
[wfmash::align] Alignment identity cutoff = 7.20e-01%
[wfmash::align] Alignment output file = /dev/stdout
[wfmash::align] time spent loading the reference index: 6.87e-03 sec
[wflign::wflign_affine_wavefront] corrupted traceback (M, but there is a mismatch) for SlycHeinz 63490000 Smel 2231274

Kindly let me know if the parameters are correct for all vs all alignment and any adjustments need to be made.

**Comment by AndreaGuarracino** [comment]
@srividya22 not soon, `ok-high-sens-wfa2` seems still broken -_-

**Comment by ekg** [comment]
And if you would please test with the current master, it'd be interesting if the problem is still there.

**Comment by AndreaGuarracino** [comment]
This has probably been solved in these months of development.

**Comment by rickbeeloo** [comment]
Hey all! 

I"m also getting this error (on master). Since we are not applying this to humans but viruses we have thousands of haplotypes. I tried this with identity (`-p`) at 95% and 99% but both resulted in this error. What does this error message exactly mean? what causes it?

**Comment by rickbeeloo** [comment]
Hey @AndreaGuarracino, thanks for the quick reply! Asked if I could share this data - will come back to this asap

**Comment by rickbeeloo** [comment]
Hey @AndreaGuarracino, that took a while... I uploaded the FASTA [here](https://filebin.net/oqb1rw1gij0l60z6)

**Comment by AndreaGuarracino** [comment]
Struggling with the download. Why is it a `tar.gz`? Can you upload the input FASTA (`.fa.` or `.fa.gz`)?

**Comment by rickbeeloo** [comment]
Yeah just checked indeed I also couldn't properly unzip it, [here](https://filebin.net/x53sbxqsmpnd5ykp) is just the fasta. Thanks for the quick replies! Curious to see what was wrong

**Comment by AndreaGuarracino** [comment]
Your FASTA seems corrupted. I had to remove the first line to be able to run `samtools faidx ...` on it.

Can you report also the output of `wfmash -v ` and the command lines that trigger the error?
